tdrs-backend/tdpservice/settings/common.py

```diff
@@ -324,16 +331,27 @@ class Common(Configuration):
 
     s3_src = "s3-us-gov-west-1.amazonaws.com"
 
-    CSP_DEFAULT_SRC = ("'none'")
+    CSP_DEFAULT_SRC = ("'self'", "*.cloud.gov", "tanfdata.acf.hhs.gov", "api-tanfdata.acf.hhs.gov")
     CSP_SCRIPT_SRC = ("'self'", s3_src)
     CSP_IMG_SRC = ("'self'", "data:", s3_src)
     CSP_FONT_SRC = ("'self'", s3_src)
-    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov")
+    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov", "tanfdata.acf.hhs.gov","api-tanfdata.acf.hhs.gov")
     CSP_MANIFEST_SRC = ("'self'")
     CSP_OBJECT_SRC = ("'none'")
     CSP_FRAME_ANCESTORS = ("'none'")
     CSP_FORM_ACTION = ("'self'")
     CSP_STYLE_SRC = ("'self'", s3_src, "'unsafe-inline'")
+    CORS_ALLOW_HEADERS = (
+        'x-requested-with',
```

Comment:
Also don't think this would be needed with us adding the cookies, but I am not sure if it worked or not.